salt/metalk8s/repo/ubuntu.sls

```diff
@@ -0,0 +1,59 @@
+{%- from "metalk8s/map.jinja" import metalk8s with context %}
+{%- from "metalk8s/map.jinja" import repo with context %}
+
+{%- set repo_host = pillar.metalk8s.endpoints['repositories'].ip %}
+{%- set repo_port = pillar.metalk8s.endpoints['repositories'].ports.http %}
+
+Install python-apt:
+  pkg.installed:
+    - name: python-apt
```

Comment:
Here you expect that some repo available before our configuration with `python-apt` available. Please remove this and just assume that `python-apt` is installed for the moment (it's installed with salt, so for the bootstrap node it's ok)